director/accounts/migrations/0018_auto_20200426_1815.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 2.2.11 on 2020-04-26 18:15
+
+from django.db import migrations, models
+
+ACCOUNT_MEMBER_NAME = "Account member"
+ACCOUNT_ADMIN_NAME = "Account admin"
+ACCOUNT_VIEWER_NAME = "Account viewer"
+
+MODIFY_PERMISSION = "modify"
+ADMINISTER_PERMISSION = "administer"
+VIEW_PERMISSION = "view"
```

Comment:
This stuff is repeated from the original migrations but it is necessary to make sure all the roles have the right permissions. Due to the way migrations are run I don't think we can just put these in a consistent place because they might not exist later.